pandas/tests/arrays/sparse/test_array.py

```diff
@@ -696,14 +694,23 @@ def test_getslice_tuple(self):
         res = sparse[
             4:,
         ]  # noqa: E231
-        exp = SparseArray(dense[4:,])  # noqa: E231
+        exp = SparseArray(
+            dense[
+                4:,
+            ]
+        )  # noqa: E231
```

Comment:
Seems the two versions of `black` disagree on how to format this...I'd say you can remove this file from this PR and we'll take a look at it later, the rest of your changes look good

Reply:
@satrio-hw actually, it should be fine to remove this comma (as with it the `noqa` comment):

```
>>> sparse[4:,]
[0.0, 5.0, nan, nan, 0.0]
Fill: nan
IntIndex
Indices: array([0, 1, 4], dtype=int32)

>>> sparse[4:]
[0.0, 5.0, nan, nan, 0.0]
Fill: nan
IntIndex
Indices: array([0, 1, 4], dtype=int32)
```